tox.ini

```diff
@@ -17,12 +17,12 @@ deps =
   pipenv
   codecov
 commands =
-  pipenv install --dev
+  pipenv install --dev --ignore-pipfile
   coverage run -m unittest
   coverage report --show-missing
 setenv =
   PYTHONPATH={toxinidir}
-
+  MASU_SECRET_KEY='603cbfcaec69ea552fc8993d673d16e7408c4e0398859d2bf95ac5a891e65d21'
```

Comment:
How secret is this? Is this something that we're okay committing to a public git repo?